app.py

```diff
@@ -109,8 +112,18 @@ def graph_refresh_solver() -> None:
     # Iterate over all registered solvers and gather packages which were not solved by them.
     for solver_name in _OPENSHIFT.get_solver_names():
         _LOGGER.info("Checking unsolved packages for solver %r", solver_name)
-        for package_name, version in _GRAPH_DB.retrieve_unsolved_python_packages(solver_name):
-            _LOGGER.info(f"Adding new package %r in version %r", package_name, version)
+        solver_info = _GRAPH_DB.parse_python_solver_name(solver_name)
+        for package_name, version, index_url in _GRAPH_DB.get_unsolved_python_package_versions(
```

Comment:
@fridex this function has _DEFAULT_COUNT set to 100, for this case, but for `graph-refresh-job` we need all unsolved. In this way, we are automatically constraining graph-refresh-job.

Reply:
Yes, that's a fair point.

In graph-refresh-job, there is a way to limit number of solvers/analyzers to be scheduled:

https://github.com/thoth-station/graph-refresh-job/blob/04feea172600dd14471559d5b8de163d3a162f61/app.py#L100-L101

We could propagate that parameter as the `count` parameter. If `count` is `None`, we don't need to set `limit` to the query.

Reply:
So I add that parameter `_THOTH_GRAPH_REFRESH_EAGER_STOP` to the function so graph-refresh-job parameter will decide in any case how many want to consider.

Reply:
OK, sounds reasonable.

Reply:
+1. It should not be set to `None` (which we allow as of now).

Reply:
... or accept count to be None, which could be better :)